beacon_node/http_api/src/lib.rs

```diff
@@ -4638,6 +4644,205 @@ pub fn serve<T: BeaconChainTypes>(
             },
         );
 
+    // POST lighthouse/database/import_blobs
+    let post_lighthouse_database_import_blobs = database_path
+        .and(warp::path("import_blobs"))
+        .and(warp::path::end())
+        .and(warp::query::<api_types::ImportBlobsQuery>())
+        .and(warp_utils::json::json())
+        .and(task_spawner_filter.clone())
+        .and(chain_filter.clone())
+        .then(
+            |query: api_types::ImportBlobsQuery,
+             blob_lists: Vec<BlobSidecarList<T::EthSpec>>,
+             task_spawner: TaskSpawner<T::EthSpec>,
+             chain: Arc<BeaconChain<T>>| {
+                task_spawner.blocking_json_task(Priority::P1, move || {
+                    if query.verify == Some(true) {
+                        for blob_list in &blob_lists {
+                            match verify_kzg_for_blob_list(blob_list.iter(), &chain.kzg) {
```

Comment:
For completeness we should also check `verify_blob_sidecar_inclusion_proof` for each blob.